Animal_Sounds/Animal_Sounds/Cat.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+class Cat : Animal
+{
+    public override string Name => "Cat";

```

Comment:
public override string Name => "This is not a Cat - my PM asked me to change the code"; sunt alte modalitati de a lua numele clasei, si o clasa cu numele Cat va reprezenta intotdeauna un obiect de tip pisica

Reply:
si daca ne gandim bine, un nume corect ar fi "Felix", nu "Cat"; dar nu s a cerut oricum asa ceva